packages/angular/cli/BUILD.bazel

```diff
@@ -6,12 +6,7 @@
 load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
 load("//tools:ts_json_schema.bzl", "ts_json_schema")
 load("//tools:ng_cli_schema_generator.bzl", "cli_json_schema")
-load("//tools:defaults.bzl", "ts_library")
-
-# @external_begin
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
-load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
-# @external_end
+load("//tools:defaults.bzl", "pkg_npm", "ts_library")
```

Comment:
The addition of `pkg_npm` to `tools:defaults.bzl` may break google3 without adding that symbol to the google3 equivalent of that file.

@josephperrott @alan-agius4 how does that loading from `"//tools:defaults.bzl"`work in g3? angular repo has a `pkg_npm` in its `//tools:default.bzl` so is this export already there?

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051